apps/cookbook/src/app/examples/section-header-example/examples/heading-with-multiline-label.ts

```diff
@@ -0,0 +1,25 @@
+import { Component } from '@angular/core';
+
+const config = {
+  selector: 'cookbook-section-header-heading-with-multiline-label',
+  template: `<kirby-section-header>
+  <kirby-label>
+    <h3 heading> Section Header with multiline label</h3>
+    <p label wrap>This label is quite lengthy, typically prone to truncation within a section header. However, with the wrap attribute applied, behold the magic of multiline functionality! Don't take my word for it; witness it yourself. It truly works. Can you believe that?</p>
```

Comment:
Consider wrapping (🥁) the wrap attribute in a code block to make it stand out:

```suggestion
    <p label wrap>This label is quite lengthy, typically prone to truncation within a section header. However, with the <code>'wrap'</code> attribute applied, behold the magic of multiline functionality! Don't take my word for it; witness it yourself. It truly works. Can you believe that?</p>
```